panel/models/time_picker.ts

```diff
@@ -0,0 +1,52 @@
+import {TimePicker as BkTimePicker, TimePickerView as BkTimePickerView} from "@bokehjs/models/widgets/time_picker"
+import type * as p from "@bokehjs/core/properties"
+
+
+
+export class TimePickerView extends BkTimePickerView {
+    declare model: TimePicker
+
+    // Override the render method or any method where time rendering is done
+    override render(): void {
+        // Fix time zone difference; add back 8 hours
+        const date = this.model.value ? new Date(this.model.value) : new Date();
+        date.setHours(date.getHours() + 8);
+
+        console.log(date);
+
+        // get type of date
+        console.log(this.model.value)
+        console.log("Type of Date:", typeof this.model.value);
+
+        console.log("Converted Date:", date.toISOString());
+        console.log("Converted Time:", date.getTime());
+        this.model.value = 72000000
```

Comment:
This breaks things for some reason.

Reply:
I would start with using development resources, so that we can get a proper stack trace. Either use `BOKEH_DEV=true` on the command line or panel's equivalent. However, this seems to be an assertion originating from third party code (`flatpickr`).